PointOfDispensingApp/src/components/VaccinationScreeningComponent.vue

```diff
@@ -94,16 +133,44 @@
 </template>
 
 <script>
+
   export default {
     name: 'VaccinationScreeningComponent',
     methods: 
     {
+     screeningChecklistUpdate()
+     {
+       if(this.isScreeningChecklistComplete())
+       {
+         this.screeningComplete = true
+       }
+       else
+       {
+         this.screeningComplete = false
+       }
+     },
+     isScreeningChecklistComplete()
+     {
+       return (this.patientInfoComfirmed && this.consentFormSigned && this.screeningCompleted && this.factSheetProvided); 
+     },
+     vaccinationProceedDecision()
+     {
+       if(this.vaccinationProceed == "Yes")
+       {
+         this.$emit('VaccinationProceed_Yes')
+       }
+       else if(this.vaccinationProceed == "No")
```

Comment:
Why are you using an `else if` here and not an `else`?

Reply:
We could also do the same return thing from above here.

Reply:
Hmm...not sure about my reasoning.  I changed it to else!